### PR TITLE
README.adoc: Add brief introduction for Delino

### LINKED ISSUE #99: Add introduction for README

### PR DESCRIPTION
**Pull request by Cherweijie**
#99

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/AddressBookParser.java [hunk 1]
```diff
@@ -42,35 +44,34 @@ public Command parseCommand(String userInput) throws ParseException {
...
 
-        case EditCommand.COMMAND_WORD:
-            return new EditCommandParser().parse(arguments);
+            case EditCommand.COMMAND_WORD:
+                return new EditCommandParser().parse(arguments);
 
-        case DeleteCommand.COMMAND_WORD:
-            return new DeleteCommandParser().parse(arguments);
+            case DeleteCommand.COMMAND_WORD:
+                return new DeleteCommandParser().parse(arguments);
```
Comment (khsc96) [inline src/main/java/seedu/address/logic/parser/AddressBookParser.java]: Probably is this switch case got to align

File: build.gradle [hunk 2]
```diff
@@ -120,22 +120,22 @@ asciidoctor {
...
+            'source-highlighter': 'coderay',
+            icons: 'font',
+            experimental: true,
+            sectlinks: true,
+            idprefix: '',  // for compatibility with GitHub preview
+            idseparator: '-',
+            'site-root': "${sourceDir}",  // must be the same as sourceDir, do not modify
+            'site-name': 'Delino',
+            'site-githuburl': 'https://github.com/AY1920S2-CS2103T-W12-1/main',
+            'site-seedu': true,  // delete this line if your project is not a fork (not a SE-EDU project)
```
Comment (khsc96) [inline build.gradle:138]: All these trailing white spaces might the issue

**Review by khsc96 — CHANGES_REQUESTED** [review]
Looks like there's some checkstyle issue please resolve, thanks!

**Review by Exeexe93 — CHANGES_REQUESTED** [review]
Travis reported that there are no newline at EOF. Please make sure that every file has a newline at the end of every file. Thanks!

File: src/main/java/seedu/address/logic/parser/AddressBookParser.java (hunk 1 above)
Comment (Cherweijie) [reply src/main/java/seedu/address/logic/parser/AddressBookParser.java]: Indentation fixed

**Review by khsc96 — APPROVED** [review]
LGTM

**Review by Exeexe93 — APPROVED** [review]
Thank for the effort to merge at your side! Look good to merge now!

**Pull request merged**